server/lib/ocr/klippa/KlippaOCRService.ts

```diff
@@ -18,8 +18,9 @@ export class KlippaOCRService implements ExpenseOCRService {
     private apiKey: string,
     private user: User,
   ) {
-    this.apiKey = apiKey;
-    this.user = user;
+    // this.apiKey = apiKey;
+    // this.user = user;
+    // See: https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd
```

Comment:
Not a fan of this TS syntax, unless you know about it it's an easy one to miss.

Suggestion for a simpler comment, in case we do want to adopt this.

```suggestion
    // Typescript constructor automatically assign parameters to `this`
```

Reply:
I understand and was also surprised by this, this is why I decided to keep the current code commented and try to explain what's happening.

Reply:
I considered keeping it like it was but "as en engineer" felt it was wrong to set something that was already there.